packages/html/src/converge.ts

```diff
@@ -0,0 +1,22 @@
+import { performance } from '@bigtest/performance';
+import { bigtestGlobals } from '@bigtest/globals';
```

Comment:
We should record work to extract interaxtor globals from bigtest globales and to use that opportunity to make them more ergonomic for things like cypress integration

Reply:
I'll file an issue for this.

Reply:
https://github.com/thefrontside/interactors/issues/3